openweathermap/weather.json

```diff
@@ -0,0 +1,58 @@
+{
+    "version": 1,
+    "request": {
+        "url": "https://community-open-weather-map.p.rapidapi.com/weather",
+        "method": "GET",
+        "authorization": "x-rapidapi-key",
```

Comment:
Hi @h-asmat, is this authorization we currently do not support? All our supported authorization methods are defined at https://github.com/sfu-db/dataprep/blob/develop/dataprep/connector/schema/defs.py#L158.

Reply:
So this is similar to how the "Bearer" API authorization is done in the Yelp config file. I played around with the Yelp and OpenWeatherMap APIs in Postman and came to the conclusion that that is how I should format it (it is a header value, x-rapidapi-key is the key, and the API key is the value). I hadn't seen the supported types until now, thank you for linking them. Will this not work then?

Reply:
If it is a header, I think here it should be HeaderAuthorization https://github.com/sfu-db/dataprep/blob/develop/dataprep/connector/schema/defs.py#L141.

Check this one for example https://github.com/sfu-db/DataConnectorConfigs/blob/more_configs/azure/bing.json

Reply:
Thank you for the resource! Didn't realize there were more config files. Changed!